synthesis/population/spatial/primary/locations.py

```diff
@@ -9,6 +9,19 @@ def configure(context):
     context.stage("synthesis.locations.work")
     context.stage("synthesis.locations.education")
 
+    context.config("education_location_source", "bpe")
+
+EDUCATION_MAPPING = {
+    "primary_school": {"min_age": 0, "max_age": 10, "type_edu": "C1"},
+    "middle_school": {"min_age": 11, "max_age": 14, "type_edu": "C2"},
+    "high_school": {"min_age": 15, "max_age": 17, "type_edu": "C3"},
+    "higher_education": {
+        "min_age": 18,
+        "max_age": 110,
+        "type_edu": ("C4", "C5", "C6"),
+    },
+}
+
```

Comment:
This is a copy & paste, please use only one spot to define all of this. You can simply do `import synthesis.population....EDUCATION_MAPPING`.

In that context, I think it could also make sense to have a distinct list of attributes somewhere. Like one file indicating all `EDUCATION_TYPE = [...]` and `EDUCATION_RANGES = [...]` and then this nice map here.